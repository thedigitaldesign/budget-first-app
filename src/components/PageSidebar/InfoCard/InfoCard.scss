@import '../../../assets/scss/variables.scss';

.info-card {
  .info-card-text {
  }
}
