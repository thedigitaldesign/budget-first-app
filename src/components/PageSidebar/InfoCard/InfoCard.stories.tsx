import React from 'react'

// Packages
import { storiesOf } from '@storybook/react'

// Components
import InfoCard from './InfoCard'

storiesOf('Components/InfoCard', module).add('Info Card', () => {
  return (
    <div>
      <InfoCard />
    </div>
  )
})
