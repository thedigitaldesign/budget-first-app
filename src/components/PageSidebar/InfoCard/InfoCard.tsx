import React, { Component } from 'react'

// Styles
import './InfoCard.scss'

export default class InfoCard extends Component {
  render() {
    return (
      <div className="info-card">
        <div className="info-card-text">
          Info Card
        </div>
      </div>
    )
  }
}
