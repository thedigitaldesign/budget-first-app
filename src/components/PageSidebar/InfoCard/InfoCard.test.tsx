import React from 'react'

// Packages
import { shallow } from 'enzyme'

// Components
import InfoCard from './InfoCard'

describe('InfoCard', () => {
  const card = shallow(<InfoCard />)

  it('renders correctly', () => {
    expect(card).toMatchSnapshot()
  })
})

