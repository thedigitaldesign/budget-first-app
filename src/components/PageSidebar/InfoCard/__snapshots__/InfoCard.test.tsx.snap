// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoCard renders correctly 1`] = `ShallowWrapper {}`;
